packages/interfaces/src/index.ts

```diff
@@ -178,6 +179,7 @@ export interface Message {
 
 export interface Encoder {
   contentTopic: string;
+  ephemeral?: boolean;
```

Comment:
```suggestion
  ephemeral: boolean;
```

The encoder will encode messages with `ephemeral` set to `true` or `false` so it does not need to be optional.

But, this does not mean that it can be optional on the constructor.